src/styles/_login.scss

```diff
@@ -0,0 +1,82 @@
+@use 'main';
+
+.bookbix-logo {
+  font-family: 'MuseoModerno', sans-serif !important;
+  color: white;
+  position: absolute;
+  padding: 20px 50px 40px 40px;
+}
+.login-header {
+  display: inline-block;
+  color: white;
+  margin-bottom: 10%;
+  font-size: medium;
+  span {
+    font-weight: 700 !important;
+    color: #2e2e2e;
+  }
+}
+.login-form {
+  &-content {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+}
+.login-left {
```

Comment:
This too fixed margin is rarely use if we want responsive design it often use fixed px on height or some of post padding for a margin for like this I prefer % or like vw